dll/win32/shell32/CDefView.cpp

```diff
@@ -592,8 +597,8 @@ CDefView::CDefView() :
     ZeroMemory(&m_FolderSettings, sizeof(m_FolderSettings));
     ZeroMemory(&m_ptLastMousePos, sizeof(m_ptLastMousePos));
     ZeroMemory(&m_Category, sizeof(m_Category));
-    m_viewinfo_data.clrText = GetSysColor(COLOR_WINDOWTEXT);
-    m_viewinfo_data.clrTextBack = GetSysColor(COLOR_WINDOW);
+    m_viewinfo_data.clrText = CLR_INVALID;
+    m_viewinfo_data.clrTextBack = CLR_INVALID;
```

Comment:
(Note: `CLR_INVALID` has the same value as `CLR_NONE`.)

Reply:
In order to understand why you set these colors to CLR_INVALID, is it done in order to always use "the most up-to-date" system colors, unless these colors somehow get fixed somewhere (and thus shouldn't change anymore when the system colors change) ?
In which scenario do the `m_viewinfo_data.clrText(Back)` get fixed to a specific value that shouldn't change?

Reply:
It gets fixed if the folder callback sets a color when it gets `SFVM_GET_CUSTOMVIEWINFO`.

Reply:
Ok this is what I was suspecting.